### DISCUSSION TITLE
MPY: soft reboot

### DISCUSSION THREAD

**Discussion by victuto** [micropython/micropython#14035] [RP2040 / Pico]
I have a problem, when I run the blink.py program, I get in the thonny shell: MPY: soft reboot. I have a raspberry pi pico W, I installed: RPI_PICO_W-20240105-v1.22.1.uf2. I had already installed other programs before and they worked well for me. Now the new programs do not work for me, the shell exits MPY: soft reboot, can you please help me solve this error.

**Comment by jimmo** [comment]
`MPY: soft reboot` is normal, that's telling the board go to back to a default state.

Do you see problems using `mpremote` ?

**Reply by ericty91** [reply]
have already solved the issue? in my case I had to erase flash mem to work round this, then flash the firmware once again
